pkg/cover/server.go

```diff
@@ -67,7 +69,7 @@ func GocServer(w io.Writer) *gin.Engine {
 	v1 := r.Group("/v1")
 	{
 		v1.POST("/cover/register", registerService)
-		v1.GET("/cover/profile", profile)
```

Comment:
why POST, not compatible with RESTful API

Reply:
"name" and "address" are long, and may have '&' in it. Use post method more suitable in this situation.